samples/deployment/vmware/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+log_path = ./ansible.log
```

Comment:
Do we really need to have the `log_path` param? By default, Ansible prints its output to the terminal where the playbook is run, so it must be sufficient for most users.

Reply:
fixed